pyaedt/edb_core/edb_data/sources.py

```diff
@@ -281,10 +281,13 @@ def net_name(self):
 
     def _create_terminal(self, is_reference=False):
         pg_term = self._edb_pin_group.GetPinGroupTerminal()
+        edb_net = self._edb_pin_group.GetNet()
+        if edb_net.IsNull():  # pragma: no cover
+            edb_net = list(self._edb_pin_group.GetPins())[0].GetNet()
```

Comment:
As we are in the `edb` module, it is not needed to prefix the name of the variable with `edb_`
So maybe it could be named:
```suggestion
        pin_group_net = self._edb_pin_group.GetNet()
        if pin_group_net.IsNull():  # pragma: no cover
            pin_group_net = list(self._edb_pin_group.GetPins())[0].GetNet()
```